openadapt/record.py

```diff
@@ -34,12 +36,34 @@
     "action": True,
     "window": True,
 }
-PLOT_PERFORMANCE = False
+PLOT_PERFORMANCE = config.PLOT_PERFORMANCE
+BYTE_TO_MB = float(2**20)
```

Comment:
@angelala3252 thank you for naming this constant. However it's still not clear to me what it represents. There are 10**6 bytes in a megabyte. Can you please clarify?

Reply:
I had a brain malfunction 😅